packages/core/src/controllers/SiweController.ts

```diff
@@ -0,0 +1,103 @@
+import { subscribeKey as subKey } from 'valtio/utils'
```

Comment:
We could probably take out some of the types and move them over to TypesUtil in core, keeping state type here only (similar to other controllers).